python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -472,42 +455,51 @@ def _get_edge_index(self, attr: CuGraphEdgeAttr) -> Tuple[TensorType, TensorType
         if attr.layout != EdgeLayout.COO:
             raise TypeError("Only COO direct access is supported!")
 
-        if isinstance(attr.edge_type, str):
-            edge_type = attr.edge_type
+        if self._is_delayed:
+            src_col_name = self.__graph.renumber_map.renumbered_src_col_name
+            dst_col_name = self.__graph.renumber_map.renumbered_dst_col_name
         else:
-            edge_type = attr.edge_type[1]
+            src_col_name = self.__graph.srcCol
+            dst_col_name = self.__graph.dstCol
```

Comment:
Right now, for SG, it always uses srcCol and dstCol as the renumbered column names.  But this may be a bad assumption in the future, along with MG always renumbering.  I will fix this.

Reply:
I opened issue #3201 and added a comment here.  Unfortunately I think that's all I can do for now until we start working on the graph creation/renumbering refactor.